contracts/validator_manager.v.py

```diff
@@ -197,7 +245,7 @@ def get_collation_header_score(shard_id: int128, collation_header_hash: bytes32)
 @constant
 def get_eligible_proposer(shard_id: int128, period: int128) -> address:
```

Comment:
Is it no longer needed?

Reply:
Correct. I will update it when I start working on notary sampling.